### PR TITLE
Add mongolian

### LINKED ISSUE #118: Add Mongolian (Монгол хэл) to supported languages
Mongolian has been fully translated and reviewed in the Editor project. 

Phonetically it should be between Maori and Nederlands.

### PR DESCRIPTION
**Pull request by chrisgarrity**
### Resolves

- Resolves #118 

### Proposed Changes

Adds Mongolian to the supported languages

## Checklist for updating translations

There are two situations in which we create manual PRs to update translations:

1. We don't want to wait for Travis's automatic weekly update; or,
2. We need to add a language that has become ready

### 1. Updating translations manually

* [x] Pull editor translations from Transifex with `> npm run pull:editor`
* [x] Pull www translations from Transifex with `> npm run pull:www`
* [x] Test the result with `> npm run test`
* [x] Confirm that you see changes to files like `editor/<resource>/<lang code>.json`

### Adding a language

* [x] Edit `src/supported-locales.js`:
  * [x] Add entry for the language in the `locales` const
  * [x] Check if language is right-to-left. If so:
    * ~Add entry in `rtlLocales`~

* [x] Check if the new language uses a country code
  * ~Check [https://www.transifex.com/explore/languages](https://www.transifex.com/explore/languages). If the language has a country code:~
 
* [x] Edit `src/index.js`:
  * [x] Add 'import' line and export line
  * [x] Add entry in `localeData` array

* [x] Check if locale is in `react-intl`
  * Look in [https://unpkg.com/react-intl/locale-data/](https://unpkg.com/react-intl/locale-data/)
  * ~If not in `react-intl`:~

* [x] Update translations per the "Updating translations" section above
* [x] Confirm that we see changes to:
    * [x] `src/supported-locales.js`
    * [x] `src/index.js`
    * [ ] ~`.tx/config` (if language needed a new locale)~
    * [x] Multiple files like `editor/<resource>/<lang code>.json`

* [x] Bump minor version number in `package.json`

* [ ] **Update scratch-blocks dependency**
    * scratch-blocks has its own translations. It will not pick up the new language unless the scratch-l10n dependency in `package.json` is updated to the new minor version.

* [x] **Add language po files to scratchr2_translations**
    * manually update `scratchr2_translations/legacy` with `tx pull -l <locale>` and check in changes

* [x] **Add language to scratchr2 settings**
    * manually update `settings/base.py` with the new language

### CODE REVIEW

**Comment by benjiwheeler** [comment]
In the checklist template, what do we mean by "export line" in the item, "Add 'import' line and export line"? I don't see a relevant "export line".

**Comment by benjiwheeler** [comment]
You plan to update the l10n version in scratch-blocks after this goes in, right?

**Comment by benjiwheeler** [comment]
A few questions, but only the one about 'Монгол хэл' in supported-locales is a potential blocker

File: src/supported-locales.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ const locales = {
     'lt': {name: 'Lietuvių'},
     'hu': {name: 'Magyar'},
     'mi': {name: 'Māori'},
+    'mn': {name: 'Монгол хэ'},
```
[the remark this answers is not in the document]
Comment (chrisgarrity) [reply src/supported-locales.js]: Yes! Must have some how deleted a character before saving. Glad you caught it.

**Comment by chrisgarrity** [comment]
Yes, I'll do the blocks one after this lands. It depends on the minor release bump to install new versions of the blocks. I think the export in the template referred to exporting `localedata`.  It's probably redundant with the next line.

**Comment by benjiwheeler** [comment]
LGTM

**Pull request merged**